prisma/schema.prisma

```diff
@@ -68,9 +68,9 @@ model VerificationToken {
 }
 
 model Company {
-  id   String @id @default(cuid())
-  name String
-
+  id                   String   @id @default(cuid())
+  name                 String
+  publicId             String
```

Comment:
inspired by https://planetscale.com/blog/why-we-chose-nanoids-for-planetscales-api